template.yaml

```diff
@@ -2,11 +2,25 @@ kind: Template
 apiVersion: v1
 template: spark
 metadata:
-  name: spark
+  name: apache-spark
+  creationTimestamp: null
+  annotations:
+    description: Apache Spark with 3 worker nodes.
+    iconClass: icon-nodejs
```

Comment:
No, not really. But at the moment, i don't know how to add icons. So i still use always the nodejs one.